Doc/whatsnew/3.11.rst

```diff
@@ -207,6 +207,32 @@ See :pep:`678` for more details.
 PEP written by Zac Hatfield-Dodds.)
 
 
+.. _whatsnew311-windows-launcher:
+
+Windows ``py.exe`` launcher improvements
+----------------------------------------
+
+The copy of the :ref:`launcher` included with Python 3.11 has been significantly
+updated. It now supports company/tag syntax as defined in :pep:`514` using the
+``-V:<company>/<tag>`` argument instead of the limited ``-<major>.<minor>``.
+This allows launching distributions other than ``PythonCore``,
+the one hosted on `python.org <https://python.org>`_.
+
+When using ``-V:`` selectors, either company or tag can be omitted, but all
+installs will be searched. For example, ``-V:OtherPython/`` will select the
+"best" tag registered for ``OtherPython``, while ``-V:3.11`` or ``-V:/3.11``
+will select the "best" distribution with tag ``3.11``.
+
+When using the legacy ``-<major>``, ``-<major>.<minor>``,
+``-<major>-<bitness>`` or ``-<major>.<minor>-<bitness>`` arguments,
+all existing behaviour should be preserved from past versions.
+Only releases from ``PythonCore`` will be selected.
```

Comment:
```suggestion
all existing behaviour should be preserved from past versions,
and only releases from ``PythonCore`` will be selected.
```
This flows better to me, and makes it clearer that the two sentences are related.